modules/weko-gridlayout/weko_gridlayout/utils.py

```diff
@@ -644,8 +645,14 @@ def get_ES_result_by_date(start_date, end_date):
     records_search = RecordsSearch()
     records_search = records_search.with_preference_param().params(
         version=False)
-    search_instance, qs_kwargs = item_search_factory(
-        None, records_search, start_date, end_date)
-    search_result = search_instance.execute()
-    rd = search_result.to_dict()
-    return rd
+    records_search._index[0] = current_app.config['SEARCH_UI_SEARCH_INDEX']
```

Comment:
Do   you use a magic number?
please define a new config = 0 and use it !

Reply:
Thank, It had been fixed.